includes/admin/pages.php

```diff
@@ -273,73 +273,76 @@ public function html()
+								<?php _e(' Google Calendar Pro add-on', 'google-calendar-events'); ?></b> <?php _e(
+	'use a different method of authorization.',
 	'google-calendar-events'
-); ?></b> <?php _e('use a different method of authorization.', 'google-calendar-events'); ?></span>
-												</div>
-												<div class="simcal-flex simcal-items-center simcal-text-gray-400 simcal-mt-[21px]">
-													<div>
-														<img src="<?php echo $admin_image_about_path . '/green-tick.png'; ?>" />
-													</div>
-													<div class="simcal-ml-[9px] simcal-text-base simcal-text-gray-400 simcal-font-normal" >
-														<span><?php _e('Display events from both private and public Google Calendars.', 'google-calendar-events'); ?></span>
-													</div>
-												</div>
-												<div class="simcal-flex simcal-items-center simcal-mt-[14px] ">
-													<div>
-														<img src="<?php echo $admin_image_about_path . '/green-tick.png'; ?>" />
-													</div>
-													<div class="simcal-ml-[9px] simcal-text-base simcal-text-gray-400 simcal-font-normal " >
-														<span> <?php _e(
-              	'Display a list of attachments with links to their original source.',
-              	'google-calendar-events'
-              ); ?></span>
-													</div>
-												</div>
-												<div class="simcal-flex simcal-items-center simcal-mt-[14px] ">
-													<div>
-														<img src="<?php echo $admin_image_about_path . '/green-tick.png'; ?>" />
-													</div>
-													<div class="simcal-ml-[9px] simcal-text-base simcal-text-gray-400 simcal-font-normal">
-														<span><?php _e('Many More...', 'google-calendar-events'); ?></span>
-													</div>
-												</div>
-												<a href="https://simplecalendar.io/addons/?utm_source=inside-plugin&utm_medium=link&utm_campaign=core-plugin&utm_content=settings-link">
-													<button type="button" class="simcal-mt-[45px] simcal-flex simcal-justify-center simcal-items-center simcal-w-[97%] simcal-h-[40px] simcal-bg-sc_green-200 simcal-text-white simcal-text-xl simcal-font-medium simcal-rounded-md">
-														<img class="simcal-p-[8px]" src="<?php echo $admin_image_about_path . '/crown.png'; ?>" />
-														<?php _e('Get Pro Version', 'google-calendar-events'); ?>
-													</button>
-												</a>
-											</div>
-										</div>
-									</div>
+); ?>
+							</span>			
+						</div>
+						<div class="simcal-pt-[29px]">
+							<a href="https://simplecalendar.io/addons/?utm_source=inside-plugin&utm_medium=link&utm_campaign=core-plugin&utm_content=settings-link">
+								<button type="button"
+									class="simcal-flex simcal-justify-center simcal-items-center simcal-w-[100%] simcal-h-[40px] simcal-bg-sc_green-200 simcal-text-white simcal-text-base simcal-font-medium simcal-rounded-md simcal-font-poppins">
+									<img class="simcal-p-[8px]" src="<?php echo $admin_image_about_path . '/crown.png'; ?>" />
```

Comment:
@sujitxtendify Please use esc_url every where you have use $admin_image_about_path make it likr  esc_url($admin_image_about_path)

Reply:
ok i will edit all the link.

Reply:
its done.